pkgs/applications/graphics/drawio/default.nix

```diff
@@ -29,13 +28,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-d8AquOKdrPQHBhRG9o1GB18LpwlwQK6ZaM1gLAcjilM=";
   };
 
-  nativeBuildInputs = [
+  nativeBuildInputs = lib.optionals (!stdenv.isDarwin) [
```

Comment:
I usually see optional dependencies at the end of the list.

Reply:
Done. There will then be rebuilds on Linux.

Reply:
Better code forever is always preferable to avoiding rebuilds once, IMO.